src/providers/MeetingProvider/types.ts

```diff
@@ -16,6 +23,7 @@ export interface MeetingJoinData {
   attendeeInfo: any;
   deviceLabels?: DeviceLabels | DeviceLabelTrigger;
   eventReporter?: EventReporter;
+  managerConfig?: ManagerConfig;
```

Comment:
Should we change this to `meetingManagerConfig`, the value and not the type, I see we already made a mistake when earlier just the ManagerConfig type was exported 😓 ?